### ISSUE TITLE
Question on  optimization method

### ISSUE THREAD

**Issue by lizhenstat** [jaehong-yoon93/CGES#6]
Hi, I noticed that the paper uses proximal gradient descent to solve the optimization problem. However the neural network is already a non-convex problem, why can't we just use the normal algorithmn (adam, etc) to solve this issue? 

Thanks in advance

**Comment by lizhenstat** [comment]
@jaehong-yoon93 Looking forward to your reply

**Comment by lizhenstat** [comment]
I asked one of my classmates and he says that if we do not use the optimization method(like proximal gradient descent), we can't achieve the sparsity.
